src/encoded/commands/write_submission_spreadsheets.py

```diff
@@ -503,12 +504,18 @@ def get_spreadsheet(item: str, submission_schema: Dict[str, Any]) -> Spreadsheet
 def get_properties(submission_schema: Dict[str, Any]) -> List[Property]:
     """Get property information from the submission schema"""
     properties = schema_utils.get_properties(submission_schema)
-    return [get_property(key, value) for key, value in properties.items()]
+    property_list = []
+    for key, value in properties.items():
+        property_list += get_property(key, value)
+    return property_list
 
 
 def get_property(property_name: str, property_schema: Dict[str, Any]) -> Property:
```

Comment:
This seems to output an array of `Property` now.

Reply:
Yes, I changed this to account for the fact that if a property is nested, it will create multiple `Property` instances, so I made them all arrays that can be concatenated together